apps/aesophia_http/src/aesophia_http_handler.erl

```diff
@@ -272,12 +272,12 @@ generate_aci(Contract, Options) ->
     end.
 
 compile_contract(Contract, Options) ->
-    Opts = compile_options(Options),
+    Opts = compile_options(Options) ++ [warn_all],
     try aeso_compiler:from_string(binary_to_list(Contract), [{aci, json} | Opts]) of
```

Comment:
before I've did it as
```suggestion
    Opts = compile_options(Options),
    try aeso_compiler:from_string(binary_to_list(Contract), [warn_all, {aci, json} | Opts]) of
```
it was looks working, but dialyzer was failing with
```
apps/aesophia_http/src/aesophia_http_handler.erl
Line 73: The pattern {'ok', ByteCode, Aci, Warnings} can never match the type {'error',binary()}
Line 76: Guard test is_list(Errors::binary()) can never succeed
Line 440: Function mk_warnings/1 will never be called
```
🤔

Reply:
Dialyzer is stupid... I think the problem is that the `option()` type in `aeso_compiler` is not complete, it doesn't think that `warn_all` is a valid option... (cc @ghallak)